contracts/mocks/MockValidator.sol

```diff
@@ -63,18 +59,18 @@ contract MockValidator is ERC7739Validator {
     // msg.sender = Smart Account
     // sender = 1271 og request sender
     function _erc1271CallerIsSafe(address sender) internal view virtual override returns (bool) {
-        return (sender == 0x000000000000D9ECebf3C23529de49815Dac1c4c || // MulticallerWithSigner
-            sender == msg.sender);
+        return (
+            sender == 0x000000000000D9ECebf3C23529de49815Dac1c4c // MulticallerWithSigner
+                || sender == msg.sender
+        );
     }
 
     function onInstall(bytes calldata data) external {
-        require(IModuleManager(msg.sender).isModuleInstalled(MODULE_TYPE_VALIDATOR, address(this), ""), "Validator is still installed");
         smartAccountOwners[msg.sender] = address(bytes20(data));
     }
 
     function onUninstall(bytes calldata data) external {
         data;
-        require(!IModuleManager(msg.sender).isModuleInstalled(MODULE_TYPE_VALIDATOR, address(this), ""), "Validator is still installed");
```

Comment:
Actually on unInstall we don't have to remove it, but onInstall was causing tests to fail because isModuleInstalled doesn't exist while the proxy constructor is being executed

Reply:
yep, cool. my question was mostly for onUninstall